README.md

```diff
@@ -728,9 +728,9 @@ values, all of which have the following fields:
 
 * `type`: One of `"string"`, `"number"`, `"boolean"`, `"null"`, `"Infinity"`, `"NaN"`, `"sequence"` or `"dictionary"`.
 
-For `"string"`, `"number"`, `"boolean"`, and `"sequence"`:
+For `"boolean"`, `"string"`, `"number"`, and `"sequence"`:
```

Comment:
👍😉

Reply:
Yeah, sorry about the reordering, I didn't catch that when I cleaned the patch up.  I'd broken 'boolean' and 'sequence' out into separate bullet points at one point then decided that was overkill, but I guess I swapped the order a bit when I put them back.

Anyway, this is the last of the issues I found, for now at least.  https://github.com/cscott/WebIDL is running now (in PHP) and generates the same output as `webidl2.js`, modulo some details about parent fields and trivia.

Reply:
Oops, I thought it's now alphabetically ordered but turns out it's not, but okay 😄

TIL the PIP library exists! I wonder one day we could build a wasm library and use it everywhere 👀

Reply:
https://github.com/cscott/node-php-embed was my contribution to "use everywhere" ;)

Seriously, we @ wikimedia are working on improving the PHP DOM libraries, so expect to see more PHP/WebIDL work in the next few months as we drag PHP's ancient libxml bindings into the modern era.